app/models/health/patient.rb

```diff
@@ -811,7 +854,7 @@ def recent_case_management_note
     # Provide a means of seeing all the case notes, regardless of data source in one location
     def case_notes_for_display
       case_notes = []
-      case_notes += client.health_touch_points.case_management_notes.order(collected_at: :desc).map do |form|
+      case_notes += client.health_touch_points.case_mancervixagement_notes.order(collected_at: :desc).map do |form|
```

Comment:
is `case_mancervixagement_notes` really the right scope?  That's a weird spelling.